htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -239,19 +248,12 @@ function print_eldy_menu($db, $atarget, $type_user, &$tabMenu, &$menu, $noout =
 				  !empty($user->rights->supplier_proposal->lire) ||
 				  !empty($user->rights->supplier_order->lire) ||
 				  !empty($user->rights->contrat->lire) ||
-				  !empty($user->rights->ficheinter->lire)
+                !empty($user->rights->ficheinter->lire ||
+					  $onlysupplierorder)
```

Comment:
Only "positive" permission should be here to decide if we show the menu.
Instead of test on 
|| $onlysupplierorder
a test on 
!empty($user->rights->fournisseur->commande->lire)
can be done ?